modal/__init__.py

```diff
@@ -37,7 +37,7 @@
     from .proxy import Proxy
     from .queue import Queue
     from .retries import Retries
-    from .sandbox import Sandbox
+    from .sandbox import Sandbox, SandboxStatus
```

Comment:
Is this enum important enough to be adding to the top-level namespace? We really want to avoid cluttering that.